src/Microsoft.TestPlatform.TestHostProvider/Hosting/DotnetTestHostManager.cs

```diff
@@ -318,7 +318,7 @@ public IEnumerable<string> GetTestPlatformExtensions(IEnumerable<string> sources
 
             if (extensions != null && extensions.Any())
             {
-                extensionPaths.AddRange(extensions.Where(x => x.EndsWith(DataCollectorRegexPattern, StringComparison.OrdinalIgnoreCase)));
+                extensionPaths.AddRange(extensions.Where(x => x.EndsWith(CoverletDataCollectorRegexPattern, StringComparison.OrdinalIgnoreCase)));
```

Comment:
@vagisha-nidhi : Do we have a work item tracking removal of this hack? We'd want coverlet to add this into the runsettings and not do any special casing, wouldn't we?

Reply:
I'll create an issue on vstest to clear up all coverlet specialized changes.

Reply:
https://github.com/microsoft/vstest/issues/2275